to_opus.py

```diff
@@ -102,12 +106,43 @@ def migrate(self):
         self.pool.close()
         self.pool.join()
 
+    def delete_converted(self):
+        self.logger.info('checking source files that do not exist anymore')
+        for root, _, files in os.walk(self.target_dir):
+            for file in files:
+                file_base, dest_ext = os.path.splitext(file)
+                if dest_ext == ".opus":
+                    dest_base = root + os.sep + file_base
+                    dest_path = dest_base + dest_ext
+                    src_base = self.source_dir + os.sep + os.path.relpath(dest_base, self.target_dir)
+                    Path(src_base).parent.mkdir(parents=True, exist_ok=True)
```

Comment:
this seems like it will leave artifacts in the source dir if a whole directory has been deleted?